framework/test/mocha/common/modules_loader.js

```diff
@@ -87,26 +86,8 @@ const modulesLoader = new function() {
 		jobsQueue.jobs = {};
 		scope = _.defaultsDeep(scope, this.scope);
 		switch (Logic.name) {
-			case 'Account':
-				new Logic(
-					scope.components.storage,
-					scope.components.logger,
-					scope.modules.rounds,
-				);
-				break;
 			case 'Block':
-				async.waterfall(
-					[
-						function(waterCb) {
-							new Account(scope.components.storage, scope.components.logger);
-
-							return waterCb();
-						},
-					],
-					() => {
-						new Logic(scope.ed, this.transactions, cb);
-					},
-				);
+				new Logic(scope.ed, this.transactions, cb);
```

Comment:
Do we need a switch statement here as there is only one real option 'Block' and the 'default'?

Reply:
Not really. Will replace by if/else ;)